src/main/java/frc/robot/commands/states/PlaceCoral.java

```diff
@@ -44,9 +50,14 @@ public void execute() {
   public void end(boolean interrupted) {
     globalCoralOuttake.setCoralOuttake(0);
     globalCoralOuttake.setHasCoralOverride(false);
+    if (globalCoralOuttake.hasCoral() == true) {
+      globalElevator.setPosition(Constants.constElevator.PREP_0);
+    } else {
+      globalElevator.setPosition(Constants.constElevator.PREP_0);
+    }
   }
```

Comment:
Good so far as an FYI, if interrupted is true that means the user has let go of the button